### DISCUSSION TITLE
Big Data

### DISCUSSION THREAD

**Discussion by Guoyfbb** [MilesCranmer/PySR#463] [Q&A]
### Feature Request

I have some questions at the moment. I am recently using pysr to perform regression on data where the input x, y, z output is temperature. However, the data volume is relatively large, with 300k rows of data, and the fitting effect is not very good. Could you please give me some advice? In addition, I would like to ask whether pysr can perform regression on partial differential operators, such as data simulated by cfd that solves ns equations. Thank you very much.

**Comment by MilesCranmer** [comment]
Hi @Guoyfbb,

300k data is too large, you probably need only 5k rows max. If you need more data then I would use `batching=true`. See tuning advice here: https://astroautomata.com/PySR/tuning/

Cheers,
Miles
